### PR TITLE
Add important notes related to localization

### PR DESCRIPTION
**Pull request by acabelloj**

### CODE REVIEW

File: desk_reference/localization.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Localization
 
+☞ Remember to use always the description field and explain as much as possible
+the context of the phrase.<br/> ☞ Each time you edit the content of a string,
```
Comment (giuband) [inline desk_reference/localization.md]: Do `<br />` tags work in markdown? I think you have to manually insert the empty lines here.

Comment (acabelloj) [reply desk_reference/localization.md]: Yes, I'll change it to the two spaces

File: desk_reference/localization.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # Localization
 
+☞ Remember to use always the description field and explain as much as possible
```
Comment (Gpx) [inline desk_reference/localization.md]: Why are we using this symbol to make a list?

Comment (acabelloj) [reply desk_reference/localization.md]: It is called `index` or `manicule` https://en.wikipedia.org/wiki/Index_(typography)

```
The typical use of the pointing hand is as a bullet-like symbol to direct the reader’s attention to important text, having roughly the same meaning as the word "attention" or "note".
```

Comment (Gpx) [reply desk_reference/localization.md]: It's a desk reference, everything is important

**Pull request merged**